package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "flick-poc",
+  "version": "1.0.0",
```

Comment:
Not a huge deal for this project, but just be aware that the version numbers typically start with "0.1.0, 0.2.0, ..." until an official release is created. Leave as is for now.